cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/util/CloudEntityResourceMapper.java

```diff
@@ -187,7 +191,11 @@ private static Date parseDate(String dateString) {
                 // then remove it before parsing
                 String isoDateString = dateString.replaceFirst(":(?=[0-9]{2}$)", "")
                     .replaceFirst("Z$", "+0000");
-                return dateFormatter.parse(isoDateString);
+                Date date = dateFormatter.parse(isoDateString);
+                long timestamp = date.getTime();
+                if (timestamp < 946684800) {
+                    logger.warn("Calculated date " + dateString + " is before 2000-1-1 considering parsed timestamp: " + timestamp);
```

Comment:
Have you tested whether this makes it into the logs of the deploy-service? We use SLF4J and here you're using Apache Commons Logging.

Reply:
It is printed in the logs, I have tested it without checking the timestamp.
Actually in that version of java-client 1.13 we use only Apache commons, there is not slf4j usage. If this is some leftover, we could change it in separate PR.